docs/src/reference-guide/components/graphql/search-filtering.md

```diff
@@ -0,0 +1,297 @@
+# Search and Filtering
+
+Currently, the Fuel indexer supports the following search and filtering operations:
```

Comment:
Suggested change:

"The Fuel indexer currently supports the following search and filtering operations:"